Extensions/Debugger/debuggertools.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tools for interracting with the debugger.
+ * @fileoverview
```

Comment:
Let's use @file for consistency with Doxygen. Also I think the description must follow the tag ;)